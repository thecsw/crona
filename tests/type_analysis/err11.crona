main : void() {
    a : int;
    a = true + "hello";
    b : bool;
    b++;
    b--;
    a = true / "hello";
    a = "one" * false;
    a = "bbbb" - true;
}