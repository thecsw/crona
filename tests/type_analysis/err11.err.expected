FATAL [3,9]: Arithmetic operator applied to invalid operand
FATAL [3,16]: Arithmetic operator applied to invalid operand
FATAL [5,5]: Arithmetic operator applied to invalid operand
FATAL [6,5]: Arithmetic operator applied to invalid operand
FATAL [7,9]: Arithmetic operator applied to invalid operand
FATAL [7,16]: Arithmetic operator applied to invalid operand
FATAL [8,9]: Arithmetic operator applied to invalid operand
FATAL [8,17]: Arithmetic operator applied to invalid operand
FATAL [9,9]: Arithmetic operator applied to invalid operand
FATAL [9,18]: Arithmetic operator applied to invalid operand
Type Analysis Failed
